packages/core/src/__tests__/state-manager.test.ts

```diff
@@ -635,13 +635,13 @@ describe('anchor', () => {
         'requestAnchor'
       )
       // Emulate CAS responses to the 1st commit
-      const fauxCASResponse$ = new Subject<AnchorServiceResponse>()
+      const fauxCASResponse$ = new Subject<CASResponse>()
       requestAnchorSpy.mockReturnValueOnce(fauxCASResponse$)
       // Subscription for the 1st commit
       const stillProcessingFirst = await ceramic.repository.stateManager.anchor(stream$)
       // The emulated CAS accepts the request
       fauxCASResponse$.next({
-        status: AnchorStatus.PENDING,
+        status: RequestStatusName.PENDING,
```

Comment:
Can we call this `AnchorRequestStatusName`?  Otherwise its not really clear reading this what the context for "Request" is.

Reply:
Yup.